src/compute-client/src/controller/instance.rs

```diff
@@ -103,13 +104,30 @@ impl From<CollectionMissing> for DataflowCreationError {
 }
 
 #[derive(Error, Debug)]
-pub(super) enum PeekError {
+#[error("the instance existed but was dropped")]
+pub(super) struct InstanceDropped();
```

Comment:
Nit: "Shut down" is more succinct than "existed but was dropped".

(Also the parentheses are unnecessary.)

```suggestion
#[error("the instance has shut down")]
pub(super) struct InstanceShutDown;
```